### DISCUSSION TITLE
Correct interpretation of ov_cdr_aff

### DISCUSSION THREAD

**Discussion by felixeych** [magpiemodel/magpie#730] [Help] [answered]
From the documentation and the declarations in the forestry module I know that the output variable ov_cdr_aff gives the expected bgc (CDR) and local bph effects of afforestation depending on planning horizon (mio. tC). Im especially interested into the bgc effects, but struggle with the interpretation of the variable. 
For each year-age-class combination, it gives a quantity of carbon sequestered in forests. Do these quantities combined represent the total carbon removed by afforestation in a given period (say, 2040) over the whole planing horizon (e.g., 50 years)? Or do they represent cdr in 2040 only, such that one would have to sum over all periods covered by the planing horizon to get total CDR?

Thanks!

**Comment by flohump** [comment, marked as answer]
The variable `ov_cdr_aff` is an intermediate variable for the re/afforestation decision-making process in MAgPIE. For each year, it contains the expected carbon removal by age-class until the end of the planning horizon. If you sum over ac in `ov_cdr_aff`, you get the total expected carbon removal for the whole the planning horizon for each year. 
`ov_cdr_aff ` is converted into annual monetary incentives in module 56_ghg_policy (`q56_reward_cdr_aff`).
If you are looking for annual gross carbon removal, I recommend to use the output variabe `Emissions|CO2|Land|Land-use Change|Regrowth|CO2-price AR (Mt CO2/yr)`. If you are looking for cumulative values: `Emissions|CO2|Land|Cumulative|Land-use Change|Regrowth|CO2-price AR (Gt CO2)`

**Reply by felixeych** [reply]
Thanks a lot!
